lib/response_bank.rb

```diff
@@ -67,6 +77,23 @@ def cache_key_for(data)
       end
     end
 
+    def check_encoding(env, default_encoding = 'br')
+      # Cloudflare now supports brotli, gzip or uncompressed
+      if env.key?('HTTP_ACCEPT_ENCODING')
+        if env['HTTP_ACCEPT_ENCODING'] == 'gzip'
+          env['response_bank.encoding'] = 'gzip'
+          env['response_bank.server_cache_encoding'] = 'gzip'
+        elsif env['HTTP_ACCEPT_ENCODING'].to_s.include?('br')
+          env['response_bank.encoding'] = 'br'
+          env['response_bank.server_cache_encoding'] = 'br'
+        end
+      else
+        # No encoding requested from client, but we still need to cache the page in server cache
+        env['response_bank.encoding'] = nil
```

Comment:
why not set these values as the default and have the `if` condition above override?